public/base-locales/de.json

```diff
@@ -281,6 +281,7 @@
   "show_ex_badge": "EX Abzeichen anzeigen",
   "search": "Suche",
   "global_search_quests": "Füge den Name der Belohnung ein...",
+  "global_search_lures": "Füge den Name des Lockmodule ein...",
```

Comment:
Namen des Lockmoduls ?

Reply:
Oh also in the other translations it uses "den Name des" 😵‍💫

Reply:
changed all